src/plugins/custom_integrations/public/components/replacement_card/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ReplacementCard } from './replacement_card';
+
+export { ReplacementCard, Props } from './replacement_card';
+export {
```

Comment:
Are these exports being used?

Reply:
Not presently.  But I tend to export them for completeness, since external consumers who might want to use the base component have to come and ask permission to export it to use it...? I can remove them for now.